hearts/hearts.py

```diff
@@ -131,6 +131,7 @@ def __init__(self, players):
         self.who_starts = None   # who_starts is not initialized until after cards are passed
 
         self.deal()
+        self.set_turn_counter()
```

Comment:
I would call this `initialize_turn_counter` to make it clear it's for the start of the game.